routes/main.js

```diff
@@ -4,14 +4,15 @@ const router = express.Router();
 const main = require("../controllers/main");
 const { fetchRandomGameData } = require("../middleware");
 const { ROUTES } = require("../controllers/routes");
+const style = require("../public/javascripts/extraStyles");
 
 router.route(ROUTES.index).get((req, res) => {
-  const extraStyles = '<link rel="stylesheet" href="/stylesheets/main.css" />';
+  const extraStyles = style.mainStyle;
```

Comment:
Same as this comment https://github.com/Leedukas/pixel-thing/pull/8/commits/91dce2cba82f659feadec6c7554af81cf9b7f036#r1160728518